src/clr-angular/progress/progress-bars/progress-bar.ts

```diff
@@ -3,17 +3,27 @@
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
  */
-import { Component, Input, HostBinding } from '@angular/core';
+import { Component, Input, HostBinding, ElementRef, AfterViewInit } from '@angular/core';
 import { isBooleanAttributeSet } from '../../utils/component/is-boolean-attribute-set';
+import { AriaLiveService } from '../../utils/a11y/aria-live.service';
 
 @Component({
+  providers: [AriaLiveService],
   selector: 'clr-progress-bar',
   template: `
     <progress [id]="id" [attr.max]="max" [attr.value]="value" [attr.data-displayval]="displayValue"></progress>
     <span *ngIf="displayAriaLive()" [attr.aria-live]="ariaLive">{{ displayValue }}</span>
   `,
 })
-export class ClrProgressBar {
+export class ClrProgressBar implements AfterViewInit {
+  constructor(private el: ElementRef, private ariaLiveService: AriaLiveService) {}
+
+  ngAfterViewInit() {
+    const textNode = this.el.nativeElement.childNodes[0];
+    if (textNode) {
+      this.ariaLiveService.announce(textNode.nodeValue, this.ariaLive);
+    }
+  }
```

Comment:
This will need a browser platform check `if (isPlatformBrowser(this.platformId)) {}`